src/healthcheckService.js

```diff
@@ -1,10 +1,14 @@
 const express = require('express')
 const os = require('os-utils')
 
+const { createLogger } = require('./logger')
+
 class HealthcheckService {
   constructor (pinning, port) {
     this.pinning = pinning
     this.port = port
+    this.logger = createLogger({ name: 'Pinning Server - HealthcheckService module' })
```

Comment:
A name doesn't have to be quite this verbose, but should lead us directly to the code in question, so I'd suggest we lean towards something shorter, like the module path, i.e. just `healthCheckService` in this case, and if it was inside a folder, it could be `folderName/moduleName`

Reply:
Sounds good, thanks mike @msterle